### ISSUE TITLE
gulp serve - Node Sass could not find a binding for your current environment 

### ISSUE THREAD

**Issue by romamon** [pnp/sp-dev-fx-webparts#1579]
**When executing "gulp serve" I get the following error:**

`PS D:\drawnbits\dev\BR_SPR_Cal\samples\react-calendar> gulp serve
Error: Missing binding D:\XXX\dev\BR_SPR_Cal\samples\react-calendar\node_modules\@microsoft\gulp-core-build-sass\node_modules\node-sass\vendor\win32-x64-64\binding.node
Node Sass could not find a binding for your current environment: Windows 64-bit with Node.js 10.x

Found bindings for the following environments:
  - Windows 64-bit with Node.js 8.x

This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass` to download the binding for your current environment.
    at module.exports (D:\XXX\dev\BR_SPR_Cal\samples\react-calendar\node_modules\@microsoft\gulp-core-build-sass\node_modules\node-sass\lib\binding.js:15:13)
    at Object.<anonymous> (D:\XXX\dev\BR_SPR_Cal\samples\react-calendar\node_modules\@microsoft\gulp-core-build-sass\node_modules\node-sass\lib\index.js:14:35)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (D:\XXX\dev\BR_SPR_Cal\samples\react-calendar\node_modules\@microsoft\gulp-core-build-sass\lib\SassTask.js:11:18)`

I tried all possible solutions I colud find:
- Removed "node_modules" folder an run npm install again
- Downgraded node to Version 10.13.0 (an reinstalled all)
- Ran "npm rebuild node-sass" with and without "force"

Sorry if this is an easy one and I'm wasting your time, but I'm quite new to this topic and stuck with this for 5 hours now. 
If anyone could help me I would be very grateful!

Thank you in advance

**Comment by Abderahman88** [comment]
Which node-version are you using now? 

(Do you have nvm installed?)

**Comment by hugoabernier** [comment]
@romamon this issue usually happens when you run a version of Node.js that's incompatible with the solution, but in your case you seem to be running a version that should work.

Try running:
```bash
npm rebuild node-sass
```

And see if it works.

If it doesn't try using [SPFx doctor](https://pnp.github.io/cli-microsoft365/cmd/spfx/doctor/) from the project folder to analyze your environment and make sure everything is compatible.

For us to be able to help you further, you'll need to fill the issue template that is provided when you open a new issue and provide us with the information we ask (e.g.: author, sample, environment, etc.). I'll go ahead and close this issue as incomplete.

Thanks for your understanding.

**Comment by romamon** [comment]
Thank you all for your answers! 
I'm running nvm with node version 10.22.0 
The version was the first thing I found here in other issues. So I tried solving my problem with a version change. Unfortunately that didn't work. 
I also tried "npm rebuild node-sass", as suggested by the the error message, but it helped neither.
I'll try SPFx doctor, thank you for the link.
And if that will not work I'll try to reinstall everything node related... maybe all that version changing / reinstalling with global / force and other parameters produced a mess, which needs to be cleaned up.
